packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Expressions/DictionaryInitializerExpression.cs

```diff
@@ -5,29 +5,33 @@
 
 namespace Microsoft.Generator.CSharp.Expressions
 {
-    public sealed record DictionaryInitializerExpression(IReadOnlyList<(ValueExpression Key, ValueExpression Value)>? Values = null) : ValueExpression
+    public sealed record DictionaryInitializerExpression(IReadOnlyDictionary<ValueExpression, ValueExpression> Values) : ObjectInitializerExpression(Values)
```

Comment:
well changing it from a list of keyvalue pair to a dictionary might make the order of writing messy, the dictionary does not preserve the order of inputs.
Would this be a problem?

Reply:
I don't know how the order of initializations would matter